tpgtools/types/property.go

```diff
@@ -471,7 +472,7 @@ func createPropertiesFromSchema(schema *openapi.Schema, typeFetcher *TypeFetcher
 		v.Title = k
 
 		p := Property{
-			title:       jsonToSnakeCase(v.Title),
+			title:       utils.JsonToSnakeCase(v.Title),
```

Comment:
```suggestion
			title:       utils.JSONToSnakeCase(v.Title),
```

nit: https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
will change